helper/linux/writejob.cpp

```diff
@@ -28,9 +28,11 @@
 #include <QDBusInterface>
 #include <QDBusUnixFileDescriptor>
 
-#include <sys/mman.h>
 #include <unistd.h>
-#include <stdlib.h>
```

Comment:
I'm not sure about the two headers above which I removed. I couldn't seem to find anything that uses them.